main.go

```diff
@@ -1,32 +1,32 @@
-// package main
-
-// import (
-// 	testserver "somas_base_platform/pkg/testServer"
-// )
-
-// func main() {
-// 	ts := testserver.New()
-// 	ts.Init()
-// 	ts.Start()
-// }
-
 package main
 
 import (
-	agent1 "somas_base_platform/pkg/agents/AgentTypes/agent1"
-	agent2 "somas_base_platform/pkg/agents/AgentTypes/agent2"
-	baseAgent "somas_base_platform/pkg/agents/BaseAgent"
+	testserver "somas_base_platform/pkg/testServer"
 )
 
 func main() {
+	ts := testserver.New()
+	ts.Init()
+	ts.RunGameLoop(1)
+}
```

Comment:
I'd much prefer if the code worked as follows:

We create the map here and call `testserver.New(map)`. Then, we no longer need to call `Init()`. As such, you should implement something like `baseServer.GetServer(map)` where the base implementation handles the conversion to an array. This should NOT be part of the interface - it is us only, not the user